### ISSUE TITLE
No live version of translated page

### ISSUE THREAD

**Issue by jkevingutierrez** [wagtail/wagtail-localize#300]
Translated pages don't have the View Live button

![image](https://user-images.githubusercontent.com/6220055/104953366-44d87380-5994-11eb-9da0-2c49deecd6f1.png)
![image](https://user-images.githubusercontent.com/6220055/104953384-4d30ae80-5994-11eb-9c4d-a4a7a437412b.png)

en version:

![image](https://user-images.githubusercontent.com/6220055/104953398-54f05300-5994-11eb-9a6f-243d0d7106d1.png)
![image](https://user-images.githubusercontent.com/6220055/104953420-5caff780-5994-11eb-9332-3944e78795e7.png)

**Comment by kaedroho** [comment]
Have you configured `i18n_patterns`? https://docs.wagtail.io/en/stable/advanced_topics/i18n.html#adding-a-language-prefix-to-urls

**Comment by jkevingutierrez** [comment]
@kaedroho yes, in fact, if I write the URL manually, the page works well, but the buttons to view the Live page aren't present as they are on the original page

**Comment by zerolab** [comment]
I believe this was fixed. Version 1.0 displays the live button when editing a translated page

**Comment by raindeer** [comment]
In my case I get the Live button for translated pages but it links to the original url only

**Comment by raindeer** [comment]
I have a similar problem but the Live button is shown but the URL is wrong. It seems to ignore the base_url. On the English original page, the Live button goes to http://localhost:8000/<slug> while on the German translation, it goes to http://localhost/de/<slug> which does not work.

It tried to set BASE_SITE_URL = 'http://localhost:8000' but that didnt help.

**Comment by kaedroho** [comment]
@raindeer this might be the site record in the admin has an incorrect port. Could you try looking in the Settings => Sites section in the admin and checking that the default site has the correct port?

**Comment by raindeer** [comment]
Thanks! that worked, but I dont understand why the English Live linked worked an not the German, shouldnt they use the same way for figuring out the URL?
